docs/src/tutorials/conic/dualization.jl

```diff
@@ -0,0 +1,218 @@
+# reformulating the problem from one form to the other, it can be more
+# efficient to solve the dual instead of the primal.
+
+# To demonstrate, we use a variation of the [Maximum cut via SDP](@ref) example.
+
+# The primal formulation (in standard conic form) is:
+
+model_primal = Model()
+@variable(model_primal, X[1:2, 1:2], PSD)
+@objective(model_primal, Max, sum([1 -1; -1 1] .* X))
+@constraint(model_primal, primal_c[i = 1:2], 1 - X[i, i] == 0)
+print(model_primal)
+
+# This problem has three scalar decision variables (the matrix `X` is symmetric),
+# two scalar equality constraints, and a constraint that `X` is positive
+# semidefinite.
+
+# The dual of `model_primal` is:
+
+model_dual = Model()
+@variable(model_dual, y[1:2])
+@objective(model_dual, Min, sum(y))
+@constraint(model_dual, dual_c, [y[1]-1 1; 1 y[2]-1] in PSDCone())
+print(model_dual)
+
+# This problem has two scalar decision variables, and a 2x2 positive
+# semidefinite matrix constraint.
+
+# !!! tip
+#     If you haven't seen conic duality before, try deriving the dual problem
+#     based on the description in [Duality](@ref). You'll need to know that the
+#     dual cone of [`PSDCone`](@ref) is the [`PSDCone`](@ref).
+
+# When we solve `model_primal` with `SCS.Optimizer`, SCS reports three variables
+# (`variables n: 3`), five rows in the constraint matrix (`constraints m: 5`),
+# and five non-zeros in the matrix (`nnz(A): 5`):
+
+set_optimizer(model_primal, SCS.Optimizer)
+optimize!(model_primal)
+Base.Libc.flush_cstdio()  #hide
+
+# (There are five rows in the constraint matrix because SCS expects problems in
+# geometric conic form, and so JuMP has reformulated the `X, PSD` variable
+# constraint into the affine constraint `X .+ 0 in PSDCone()`.)
+
+# The solution we obtain is:
+
+value.(X)
+
+#-
+
+dual.(primal_c)
+
+#-
+
+objective_value(model_primal)
+
+# When we solve `model_dual` with `SCS.Optimizer`, SCS reports two variables
+# (`variables n: 2`), three rows in the constraint matrix (`constraints m: 3`),
+# and two non-zeros in the matrix (`nnz(A): 2`):
+
+set_optimizer(model_dual, SCS.Optimizer)
+optimize!(model_dual)
+Base.Libc.flush_cstdio()  #hide
+
+# and the solution we obtain is:
+
+dual.(dual_c)
+
+#-
+
+value.(y)
+
+#-
+
+objective_value(model_dual)
+
+# The problems are small enough that it isn't meaningful to compare the solve
+# times, but in general, we should expect the dual problem to solve faster
+# because it contains fewer variables and constraints. The difference is
+# particularly noticeable on large-scale optimization problems.
+
+# ## `dual_optimizer`
+
+# Manually deriving the conic dual is difficult and error-prone. The package
+# [Dualization.jl](@ref) provides the `Dualization.dual_optimizer` meta-solver,
+# which wraps any MathOptInterface-compatible solver in an interface that
+# automatically formulates the dual of an input problem, solves the dual
+# problem, and then reports the primal solution back to the user.
```

Comment:
"reports the primal solution back to the user." is confusing, one might think you mean that Dualization reports the primal solver solution to the solver as primal solution without transforming it to dual solution